### PR TITLE
Adding readme files 

### PR DESCRIPTION
**Pull request by Noelryn**
# Github Demo

* adding lots of readme with oliver

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,8 @@ Git demo
 
 Hi Oliver
 
+Hi Noel!
+
```
Comment (oliverochman) [inline README.md]: Remove empty lines

**Review by oliverochman — CHANGES_REQUESTED** [review]
Please change this so I can merge it

File: README.md (hunk 1 above)
Comment (Noelryn) [reply README.md]: thanks I will change this

**Review by oliverochman — APPROVED** [review]

**Review by oliverochman — APPROVED** [review]
LGTM!

**Pull request merged**